tests/cli/test_rasa_data.py

```diff
@@ -152,6 +153,87 @@ async def mock_from_importer(importer: TrainingDataImporter) -> Validator:
         data.validate_files(args)
 
 
+def test_rasa_data_augment_nlu(run: Callable[..., RunResult],):
+
+    out_path = "data/augmentation_results"
```

Comment:
how about using a `tmp_path` to avoid these results interfering with other tests and to make sure the data is removed after running the test?